### DISCUSSION TITLE
Parking this here: Responsive Table in ReactJS with TailwindCSS

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#3021] [General]
Hi

I've just found this and parking it here as I need to use it with DaisyUI in a vanilla javascript project.

https://www.tailwindtap.com/components/table/mobile-responsive-table

**Comment by saadeghi** [comment]
You can open their [preview link](https://www.tailwindtap.com/components/table/view/table-mobile-responsive-table)  
Copy the `<table>` tag from devtools and use it as HTML: https://play.tailwindcss.com/EPNr4WnzhK
